src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -233,7 +233,7 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport with ElasticS
     results map { suggestResult =>
       val compSugg: CompletionSuggestion = suggestResult.getSuggest.getSuggestion(suggestionName)
       val options : Seq[CompletionSuggestion.Entry.Option] = compSugg.getEntries.get(0).getOptions.asScala
-      options map { option: CompletionSuggestion.Entry.Option => option.getText.string }
+      (options map { option: CompletionSuggestion.Entry.Option => option.getText.string }).distinct
```

Comment:
```
options.map(_.getText.string).distinct
```
reads easier to me in this case.

Reply:
I took the `map` suggestion - good call!

This code doesn't lend well to a for comp, because it mixes future resolution and non-futures, as well as mixing Scala and Java (the ES API is pure Java), so I didn't rewrite it.